src/database/migrations/20190901120620-adding_to_user.js

```diff
@@ -0,0 +1,46 @@
+export default {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.addColumn('Users', 'gender', {
+      type: Sequelize.STRING,
+      allowNull: true
+    });
+    await queryInterface.addColumn('Users', 'birthday', {
+      type: Sequelize.STRING,
+      allowNull: true
+    });
+    await queryInterface.addColumn('Users', 'preferredlanguage', {
+      type: Sequelize.STRING,
+      allowNull: true
+    });
+    await queryInterface.addColumn('Users', 'currency', {
+      type: Sequelize.STRING,
+      allowNull: true
+    });
+    await queryInterface.addColumn('Users', 'residentialaddress', {
+      type: Sequelize.STRING,
+      allowNull: true
+    });
+    await queryInterface.addColumn('Users', 'role', {
+      type: Sequelize.STRING,
+      allowNull: true
+    });
+    await queryInterface.addColumn('Users', 'department', {
+      type: Sequelize.STRING,
+      allowNull: true
+    });
+    await queryInterface.addColumn('Users', 'linemanager', {
+      type: Sequelize.STRING,
```

Comment:
The linemanager is going to be a user in the app too so we should save the linemanager's user id and not the name. You will need to change the type here to INTEGER